### PR TITLE
[bitnami/redis] Added extraEnvVars for metrics container

### PR DESCRIPTION
**Pull request by abhi1693**
Signed-off-by: Abhimanyu Saharan <desk.abhimanyu@gmail.com>

**Description of the change**

Added extraEnvVars for metrics container

**Benefits**

The exporter has several options which can be needed to change the defaults.

**Possible drawbacks**

NA

**Applicable issues**

  - fixes #

**Additional information**

**Checklist**

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the values.yaml and added to the README.md using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/redis/Chart.yaml [hunk 1]
```diff
@@ -24,4 +24,4 @@ maintainers:
 name: redis
 sources:
   - https://github.com/bitnami/bitnami-docker-redis
-version: 16.5.4
+version: 16.5.5
```
Comment (javsalgar) [inline bitnami/redis/Chart.yaml:27]: Could you bump the minor version? It is adding a non-breaking feature

**Review by javsalgar — APPROVED** [review]
LGTM but check my comment

File: bitnami/redis/Chart.yaml (hunk 1 above)
Comment (abhi1693) [reply bitnami/redis/Chart.yaml:27]: Just for future reference, when do you guys prefer the patch version change?

**Comment by abhi1693** [comment]
@javsalgar Is there any ETA on the merge and deployment for this? I have a deployment pending for our production clusters and without this change, our metrics will not work.

**Review by javsalgar — APPROVED** [review]

**Comment by javsalgar** [comment]
Merged! Thank you so much!

**Pull request merged**